assets/chef-recipes/cookbooks/mariadb/recipes/mdberepos.rb

```diff
@@ -16,14 +16,14 @@
   # Split MaxScale repository information into parts
   if node['mariadb']['repo'].include?('es-repo.mariadb.net')
     file "/etc/apt/sources.list.d/#{repo_file_name}.list" do
```

Comment:
Use apt_repository instead of the file resource to manage repositories. https://docs.chef.io/resources/apt_repository/